language/move-compiler/src/typing/core.rs

```diff
@@ -72,7 +72,7 @@ pub struct Context<'env> {
     pub current_function: Option<FunctionName>,
     pub current_script_constants: Option<UniqueMap<ConstantName, ConstantInfo>>,
     pub return_type: Option<Type>,
-    locals: UniqueMap<Var, Type>,
+    locals: BTreeMap<Var, Type>,
```

Comment:
Just for my own information -- why the switch away from `UniqueMap` here?  IIUC, `Var`s will still be unique coming into this mapping.

Reply:
I was being lazy and didn't add TName for Var. I should really do that